File: HeartbitGamesNet/Models/OrderDetailRespAdmin.cs
namespace HeartbitGamesNet.Models
{
    public class OrderDetailRespAdmin
    {
        public string VideogameIds { get; set; }
        public int Quantity { get; set; }
        public string FullName { get; set; }
        public string Address { get; set; }
        public string Country { get; set; }
        public string State { get; set; }
        public string ZipCode { get; set; }
        public string CardNumber { get; set; }
        public int CCV { get; set; }
        public string CardOwner { get; set; }
        //In order to send only one item in the response, we will hold the data of the videogames inside this order detail response object
        public List<OrderVideogame> OrderVideogames { get; set; }
    }
}

File: cliente/src/app/videogame-order/types/OrderDetailRespAdmin.ts
export type OrderDetailRespAdmin = {
    videogameIds?:number,
    quantity?:number,
    fullName?:string,
    address?:string,
    country?:string,
    state?:string,
    zipCode?:string,
    cardOwner?:string,
    cardNumber?:string,
    ccv?:number,
    orderVideogames?:OrderDetailVideogameResp[]
}

export type OrderDetailVideogameResp = {
    id?:number,
    title?:string,
    price?:number,
    quantity?:number
}